src/sorcha/ephemeris/simulation_parsing.py

```diff
@@ -39,7 +50,18 @@ def parse_orbit_row(row, epoch, ephem, sun_dict, gm_sun):
                 row["inc"] * np.pi / 180.0,
                 row["node"] * np.pi / 180.0,
                 row["argPeri"] * np.pi / 180.0,
-                epoch - (row["ma"] * np.pi / 180.0) * np.sqrt(row["a"] ** 3 / gm_sun),
+                epoch - (row["ma"] * np.pi / 180.0) * np.sqrt(row["a"] ** 3 / GMSUN),
+                epoch,
+            )
+        elif orbit_format == "BKEP":
+            ecx, ecy, ecz, dx, dy, dz = universal_cartesian(
+                gm_total,
+                row["a"] * (1 - row["e"]),
+                row["e"],
+                row["inc"] * np.pi / 180.0,
+                row["node"] * np.pi / 180.0,
+                row["argPeri"] * np.pi / 180.0,
+                epoch - (row["ma"] * np.pi / 180.0) * np.sqrt(row["a"] ** 3 / GMSUN),
```

Comment:
Shouldn't this be `gm_total` instead of `GMSUN` for the mean anomaly to time of perihelion passage conversion?

Reply:
Yes, good catch, @bernardinelli !